### PR TITLE
fix wording

### PR DESCRIPTION
**Pull request by SummerSun**

### CODE REVIEW

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java [hunk 1]
```diff
@@ -36,7 +36,8 @@
     public static final String START_APP = "Starting Web App after deploying artifacts...";
     public static final String STOP_APP_DONE = "Successfully stopped Web App.";
     public static final String START_APP_DONE = "Successfully started Web App.";
-    public static final String WEBAPP_NOT_EXIST_FOR_SLOT = "Please configure an existing web app for slot deployment.";
+    public static final String WEBAPP_NOT_EXIST_FOR_SLOT = "The web app specified in pom.xml does not exist. " +
```
Comment (jdneo) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java]: web app -> Web App

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java [hunk 2]
```diff
@@ -36,7 +36,8 @@
     public static final String START_APP = "Starting Web App after deploying artifacts...";
     public static final String STOP_APP_DONE = "Successfully stopped Web App.";
     public static final String START_APP_DONE = "Successfully started Web App.";
-    public static final String WEBAPP_NOT_EXIST_FOR_SLOT = "Please configure an existing web app for slot deployment.";
+    public static final String WEBAPP_NOT_EXIST_FOR_SLOT = "The web app specified in pom.xml does not exist. " +
+            "Please configure an existing one for slot deployment.";
```
Comment (jdneo) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java]: Please make sure the Web App name is correct.

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java [hunk 3]
```diff
@@ -18,22 +18,23 @@
...
-            "Need configure it in pom.xml for deploying it to deployment slot.";
+    private static final String INVALID_SLOT_SETTINGS = "No <deploymentSlotSetting> is specified in pom.xml. " +
+            "Please configure it for slot deployment.";
     private static final String SLOT_NAME_PATTERN = "[A-Za-z0-9-]{1,60}";
-    private static final String INVALID_SLOT_NAME = String.format("<slotName> must match %s", SLOT_NAME_PATTERN);
+    private static final String INVALID_SLOT_NAME = String.format(
+            "Invalid <slotName> in pom.xml, it need to match the pattern '%s'", SLOT_NAME_PATTERN);
     private static final String EMPTY_CONFIGURATION_SOURCE =
-            "Creating a brand new deployment slot without any configuration.";
+            "Start creating a brand new deployment slot without any configuration...";
```
Comment (jdneo) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java]: Start creating -> Creating

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java [hunk 4]
```diff
@@ -18,22 +18,23 @@
...
     private static final String DEFAULT_CONFIGURATION_SOURCE =
-            "Creating deployment slot and copy configuration from parent.";
+            "Start creating a deployment slot and copy configuration from parent...";
```
Comment (jdneo) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java]: Start creating -> Creating

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java [hunk 5]
```diff
@@ -18,22 +18,23 @@
...
     private static final String COPY_CONFIGURATION_FROM_SLOT =
-            "Creating deployment slot and copy configuration from slot '%s'";
-    private static final String CREATE_DEPLOYMENT_SLOT = "Start create deployment slot...";
+            "Start creating a deployment slot and copy configuration from slot '%s'...";
```
Comment (jdneo) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java]: Start creating -> Creating

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java [hunk 6]
```diff
@@ -18,22 +18,23 @@
...
     private static final String CREATE_DEPLOYMENT_SLOT_DONE = "Successfully created deployment slot.";
     private static final String UNKNOWN_CONFIGURATION_SOURCE =
-            "Unknown <configurationSource> value. Please use 'NEW', 'PARENT' or other slot name.";
+            "Unknown <configurationSource> value for creating deployment slot. " +
+                    "Please use 'NEW', 'PARENT' or other deployment slot's name.";
     private static final String TARGET_CONFIGURATION_SOURCE_SLOT_NOT_EXIST =
-            "The slot configured in <configurationSource> to copy configuration from does not exist.";
+            "The deployment slot specified in <configurationSource> to copy configuration from does not exist.";
```
Comment (jdneo) [inline azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java]: Just `The deployment slot specified in <configurationSource> does not exist.` is fine

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java [hunk 7]
```diff
@@ -36,20 +36,17 @@
...
+    public static final String WEBAPP_NOT_EXIST_FOR_SLOT = "The Web App specified in pom.xml does not exist. " +
+            "Please make sure the Web App name is correct.";
     public static final String SLOT_SHOULD_EXIST_NOW = "Target deployment slot still does not exist." +
             "Please check if any error message during creation";
 
     protected DeploymentUtil util = new DeploymentUtil();
 
     @Override
     protected void doExecute() throws Exception {
-        getLog().info(String.format(WEBAPP_DEPLOY_START, getAppName()));
```
[the remark this answers is not in the document]
Comment (SummerSun) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java]: Yes, we care.

The "Start deploy.." message gets output before "Updating web app" or "Creating web app", the change moves the output message to when deploy really starts.

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/DeployTargetType.java [hunk 8]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.webapp.configuration;
```
[the remark this answers is not in the document]
Comment (SummerSun) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/configuration/DeployTargetType.java]: Those changes are required to fix the deploy related sentences, whether we are deploying to "Web App" or "Deployment Slot".

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java [hunk 9]
```diff
@@ -18,22 +18,23 @@
 import java.util.regex.Pattern;
 
 public class DeploymentSlotHandler {
-    private static final String INVALID_SLOT_SETTINGS = "<deploymentSlotSetting> is NULL." +
-            "Need configure it in pom.xml for deploying it to deployment slot.";
+    private static final String INVALID_SLOT_SETTINGS = "No <deploymentSlotSetting> is specified in pom.xml. " +
```
[the remark this answers is not in the document]
Comment (SummerSun) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java]: Now are putting it at the end of the last line.

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java [hunk 10]
```diff
@@ -18,22 +18,23 @@
...
     private static final String SLOT_NAME_PATTERN = "[A-Za-z0-9-]{1,60}";
-    private static final String INVALID_SLOT_NAME = String.format("<slotName> must match %s", SLOT_NAME_PATTERN);
+    private static final String INVALID_SLOT_NAME = String.format(
+            "Invalid <slotName> in pom.xml, it need to match the pattern '%s'", SLOT_NAME_PATTERN);
     private static final String EMPTY_CONFIGURATION_SOURCE =
-            "Creating a brand new deployment slot without any configuration.";
+            "Creating a brand new deployment slot without any configuration...";
     private static final String DEFAULT_CONFIGURATION_SOURCE =
-            "Creating deployment slot and copy configuration from parent.";
+            "Creating a new deployment slot and copy configuration from parent...";
```
[the remark this answers is not in the document]
Comment (SummerSun) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java]: use 'Creating and coping'

File: azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java (hunk 9 above)
[the remark this answers is not in the document]
Comment (akaroml) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java]: Make sure #253 is actually addressed. Thanks.

[the remark this answers is not in the document]
Comment (SummerSun) [reply azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandler.java]: Sure

**Pull request merged**